Simplenote/Classes/SPNoteListViewController+Extensions.swift

```diff
@@ -622,6 +705,40 @@ extension SPNoteListViewController {
             notesListController.object(at: indexPath) as? Note
         }
     }
+
+    open override func setEditing(_ editing: Bool, animated: Bool) {
+        super.setEditing(editing, animated: animated)
+        refreshEditButtonTitle()
+        
```

Comment:
I'm getting a `Trailing Space Violation` at this spot. No local changes. Mind confirming?